### PR TITLE
Documentation+Kernel/aarch64: Add proper support for the Raspberry Pi 4B

### PR DESCRIPTION
**Pull request by caoimhebyrne**
We kind-of supported the Raspberry Pi 4B already, but there was just a change to the MMU that was needed.

With these changes, we reach the same stage as the 3 :)
image

### CODE REVIEW

**Review by supercomputer7 — APPROVED** [review]
Looks good to me!
I have a RPi4 too, so I might try this some time soon too :)

File: Documentation/RunningOnRaspberryPi.md [hunk 1]
```diff
@@ -48,6 +48,13 @@ This step is needed because the original firmware files need to be present on th
 
 Please follow one of the existing guides (for example [here](https://scribles.net/setting-up-serial-communication-between-raspberry-pi-and-pc)) and make sure UART is working on Raspberry Pi OS before proceeding.
 
+If you're using a Raspberry Pi 4 and want to test if the UART is working correctly, you need to do a few extra steps.
```
Comment (nico) [inline Documentation/RunningOnRaspberryPi.md:51]: I think this i sonly for a 4B. The normal 4 doesn't have bt.

(i.e. s/4/4B` here)

**Review by nico — APPROVED** [review]
Thanks!

**Comment by caoimhebyrne** [comment]
Fixed, @nico!

**Review by BertalanD — APPROVED** [review]

**Pull request merged**